clientd3d/util.c

```diff
@@ -188,6 +188,10 @@ void CenterWindow(HWND hwnd, HWND hwndParent)
    RECT rcDlg, rcParent, rcScreen;
    int x, y;
 
+   // If the window to center has no parent, set hwndParent to the main window

```

Comment:
This comment doesn't add anything useful---it just says in English exactly what the code does.  What the reader wants to know is *why* this check is here.